app/routes/ticket.js

```diff
@@ -0,0 +1,152 @@
+const express = require('express')
+const router = express.Router()
+const auth = require('../middleware/auth')
+const { isValidObjectId } = require('../utils/utils')
+const ticketController = require('../controllers/ticket')
+const isUnderMaintenance = require('../middleware/maintenance')
+
+// CREATE A TICKET
+router.post(\
+  '/', 
+  isUnderMaintenance, 
+  auth, 
+  ticketController.create
+)
+
+// GET TICKETS - ALL OR FILTERED
+router.get(
+  '/', 
+  isUnderMaintenance, 
+  auth, 
+  ticketController.getTicket
+)
+
+// EDIT A TICKET BY ID
+router.put(
+  '/:id', 
+  isUnderMaintenance, 
+  auth, 
+  isValidObjectId, 
```

Comment:
@ksraj123 You have used it as middleware. It's not utility.
Also from my point of view it should not be used as middleare it should be used as a utility function inside your defination functions